tools/codegen/pkg/proc_interface_gen/generator.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interfacegen
+
+import (
+	"io/ioutil"
+	"os"
+	"text/template"
+
+	"fmt"
```

Comment:
this file needs a `bin/fmt.sh` fix. we have standard lib imports all over this block.